### ISSUE TITLE
Failed API call for chromatin accessibility

### ISSUE THREAD

**Issue by YingX97** [fabilab/cell_atlas_approximations_API#2]
await api.average('h_sapiens','chr1:9955-10355','Lung',null,'chromatin_accessibility')
{ error: 'API call failed: undefined' }
